content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,315 @@
-==============================================================
-Create alternative requests for quotation for multiple vendors
-==============================================================
+==================
+Alternative |RfQs|
+==================
```

Comment:
While the |shorthand| does work in the H1, I'd like to leave it out just in case it doesn't format properly :)
```suggestion
================
Alternative RfQs
================
```

Reply:
Suggestion implemented ✅